CMakeLists.txt

```diff
@@ -84,6 +85,35 @@ if (NOT NoLottie)
     target_link_libraries(telegram-tdlib PRIVATE rlottie)
 endif (NOT NoLottie)
 
+if (NOT NoTranslations)
+    find_package(Gettext REQUIRED)
+    # Can't use GETTEXT_CREATE_TRANSLATIONS: Overwrites pofiles.
+    # Don't want to use GETTEXT_PROCESS_POT_FILE: Expects pofiles in bindir.
+    # Can't use GETTEXT_PROCESS_PO_FILES: Would require separate calls.
+    # So we have to roll our own.
```

Comment:
This looks like it works: https://github.com/ars3niy/tdlib-purple/commit/6aff5ac9c8c069449df003ed53caa486b742b66c

Are there any problems there?

(I'm going to merge https://github.com/ars3niy/tdlib-purple/tree/translation eventually).

Reply:
Hmm. You're right, it is better. :)

In case you want to know my thought process: When reading through FindGettext.cmake, I got to the conclusion that `GETTEXT_PROCESS_PO_FILES` needs to be called once for each po-file anyway, and at that point I can just call `msgfmt` directly myself. I hadn't considered that `GETTEXT_PROCESS_PO_FILES` also takes care of the dependencies and actually installing the files.

In case you don't: https://www.reddit.com/r/puppies :P